### ISSUE TITLE
Recovering rescored logits

### ISSUE THREAD

**Issue by OllieBroadhurst** [kensho-technologies/pyctcdecode#83]
Hi there!

I was wondering if it would at all be possible to recover the updated model logits after the decoder has rescored them? From what I can tell, the logits themselves are "[lost](https://github.com/kensho-technologies/pyctcdecode/blob/a477d796e232b476ee8b877efba98aa2d822232e/pyctcdecode/decoder.py#L499)" during decoding so I have my doubts.

The use case is a bit of an edge case, but I'd like to use the updated logits for teacher-student training where the teacher model has benefitted from the LM beforehand.

Thanks!
